src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -5409,13 +5517,6 @@ namespace Microsoft.FSharp.Core
            when ^T : byte        = RangeByte    (retype start) (retype step) (retype finish)
         
 
-        type ``[,]``<'T> with 
```

Comment:
To note: We determined this was not used anywhere and is also not public. Effectively, dead code.